### PR TITLE
added translations

### PR DESCRIPTION
**Pull request by PwccaCode**
more todo later

### CODE REVIEW

File: EDEngineer/Resources/Data/localization.json [hunk 1]
```diff
@@ -4828,7 +4828,7 @@
     },
     "Super Capacitor": {
       "es": "Supercondensador",
-      "pt": null,
+      "pt": "Supercapacitor,
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json:4831]: you're missing a double quote here

File: EDEngineer/Resources/Data/localization.json [hunk 2]
```diff
@@ -4900,7 +4900,7 @@
     },
     "Drag Drives": {
       "es": "Impulsores de Arrastre",
-      "pt": null,
+      "pt": "Motores de Arrasto,
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json:4903]: same here

File: EDEngineer/Resources/Data/localization.json [hunk 3]
```diff
@@ -4909,7 +4909,7 @@
     },
     "Super Conduits": {
       "es": "Superconductores",
-      "pt": null,
+      "pt": "SuperConduites,
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json:4912]: same here and some more below

**Review by msarilar — CHANGES_REQUESTED** [review]
thanks a lot! a few double quotes missing before this can be merged

**Comment by PwccaCode** [comment]
is there anyway i can check myself so that dont happen in the future?

**Comment by msarilar** [comment]
> is there anyway i can check myself so that dont happen in the future?

yeah if you look at the file changed here https://github.com/msarilar/EDEngineer/pull/554/files you can see github highlights the parts that are bad

I thought CI would run for each pull request, a test would fail with json missing double quotes

File: EDEngineer/Resources/Data/localization.json [hunk 4]
```diff
@@ -5089,7 +5089,7 @@
     },
     "AX Small Calibre Munitions": {
       "es": "Munición AX de Calibre Pequeño",
-      "pt": null,
+      "pt": "Munição AX de Baixo Calibre,
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json:5092]: missing parenthesis

**Pull request merged**